docs/guides/tutorials.rst

```diff
@@ -5,20 +5,23 @@
 Tutorials
 #########
 
-These tutorials will teach you everything you need to know to get up and running with Great Expectations.
+These tutorials will teach you the basics of what you need to know to get up and running with Great Expectations.
 
-If you're the inpatient type, head to :ref:`Quick start` to get going with no fuss or explanation.
+* If you're the inpatient type, head to :ref:`tutorials__quick_start` to get going with no fuss or explanation.
 
-If you're a new user of Great Expectations, check out :ref:`_tutorials__getting_started` to learn important concepts by setting your first deployment.
+* If you want to simply **explore** Expectations in a notebook with some sample data without setting up a Data Context, hop over to the :ref:`tutorials__explore_expectations_in_a_notebook` tutorials.
 
-If you're interested in contributing to the open source project, thank you! Please check out :ref:`Contribute to Great Expectations` to learn how.
+* If you're a **new user** of Great Expectations, check out :ref:`tutorials__getting_started` to learn important concepts by setting up your first local deployment of Great Expectations. **We recommend you get started here!**
+
+* If you already have a working deployment of Great Expectations and want a deep dive into how to create and edit Expectations, go to the :ref:`tutorials__create_expectations` tutorial.
+
+* Finally, if you're interested in contributing to the open source project, thank you! Please check out :ref:`the Contributing section <contributing>` to learn how.
 
 
 .. toctree::
-   :maxdepth: 2
+   :hidden:
```

Comment:
Well, that's a handy directive to know...

Reply:
It took me a while to understand why we have these awkwardly placed TOCs at the bottom of each page :) This way you can still define the child tree structure but the TOC isn't visible if it's not needed. I figured we could slowly overhaul other pages like this as we encounter them.

Reply:
❤️